### DISCUSSION TITLE
Visualisation - List index out of range

### DISCUSSION THREAD

**Discussion by a-m-a-n-d-a** [asreview/asreview#1007] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Hi, 

I am encountering some problems with visualising the h5 files.  I am using the following code for visualisation: 

![asreview visualisation](https://user-images.githubusercontent.com/69677730/159159128-f1441d0b-26e2-428b-b177-8783a1a818dc.jpg)

While I am generally able to produce the visuals for most h5 files with no issues, there are some h5 files that presents an index error.

![list index out of range](https://user-images.githubusercontent.com/69677730/159158999-f9f4c412-bb66-4272-ad55-91b3135181aa.jpg)

May I know how should I go about fixing this problem? Thank you.

**Comment by Rensvandeschoot** [comment, marked as answer]
Would you be willing to share a `.h5` file that causes problems so that we can try to replicate the issue?

**Reply by a-m-a-n-d-a** [reply]
Apologies for the late reply. 

I found out the reason why these .h5 files are causing problems. It seems that the h5 files that are corrupted due various reaons (e.g. the active learning process was interrupted due to a lack of memory) are the files that produce this error message.

**Reply by Rensvandeschoot** [reply]
Thanks for letting us know!
